man/systemd.network.xml

```diff
@@ -815,6 +815,12 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
           the wire and have them accepted properly. When unset, the kernel's default will be used.
           </para></listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>IPv4RouteLocalnet=</varname></term>
+          <listitem><para>Takes a boolean. When true, the kernel does not consider loopback addresses as martian source or destination
+          while routing. This enables the use of 127/8 for local routing purposes. When unset, the kernel's default will be used.
```

Comment:
`127/8` -> `127.0.0.0/8`
kernel's ip-sysctl.txt says so, but please do not abbreviate the address.